packages/playwright/src/reporter.ts

```diff
@@ -340,22 +350,37 @@ export default class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
-        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-browser");
 
-        const output: string[] = [];
+      const output: string[] = [];
+
+      const executedProjectWithReplay = !Object.keys(this._projects).some(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && usingReplay;
+      });
+
+      if (!executedProjectWithReplay) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-project");
         output.push(emphasize("None of the configured projects ran using Replay Chromium."));
-        if (!existsSync(getRuntimePath())) {
+      }
+
+      if (executedProjectWithReplay && !existsSync(getRuntimePath())) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-browser");
```

Comment:
> Also should this be a warning or an error? I guess warning is okay?

It depends. If no projects with Replay were selected then this could be a warning - otherwise an error. I don't feel like complicating it this way right now. It wouldn't be thrown here anyway.